introduction.md

```diff
@@ -0,0 +1 @@
+Hello! I am here to teach people about GitHub.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
okay codeschool-kiddo!  BTW, are you in Florida?  I hope you are safe!!  I like all of the code school paths so far.  This one though, GitHub, confuses me so.  But ... onward!  Thanks, Mary Jane Graff aka majorg38